rskj-core/src/main/java/co/rsk/mine/MinerManager.java

```diff
@@ -18,14 +18,13 @@
 
 package co.rsk.mine;
 
-import org.ethereum.core.Blockchain;
 
 /**
  * Created by ajlopez on 15/04/2017.
  */
 public class MinerManager {
-    public void mineBlock(Blockchain blockchain, MinerClient minerClient, MinerServer minerServer) {
-        minerServer.buildBlockToMine(blockchain.getBestBlock(), false);
+    public void mineBlock(MinerClient minerClient, MinerServer minerServer) {
```

Comment:
Isn't this a good time to get rid of this useless class? Or give it some responsibilities.

Reply:
I did reduce the number of parameters on the signature of `mineBlock` which I consider a step forward in getting rid of this class. I don't think this PR is the place to remove it. 
Since I agree it should be removed, I'll create an issue for doing that after this PR is merged to master.

Reply:
Thanks!